moduleroot/spec/spec_helper.rb

```diff
@@ -32,6 +32,14 @@ def exclude_test_os
   end
 end
 
+# Use the above environment variables to limit the platforms under test
+def on_os_under_test
+  on_supported_os.reject do |os, facts|
+    (only_test_os() and not only_test_os.include?(os)) or
```

Comment:
I copied it from our spec files which already include ```and``` and ```not```, but I'll change it.